<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientGraphique</class>
 <widget class="QWidget" name="ClientGraphique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tarot Virtuel</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="layout_moi">
     <item>
      <widget class="QLabel" name="label_moi">
       <property name="text">
        <string>Vous jouez en tant que : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="LabelNom" name="nom_joueur">
       <property name="text">
        <string>???</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="layout_principal">
     <item>
      <widget class="QGroupBox" name="groupe_mes_cartes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Mes cartes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="ListeCartes" name="mes_cartes"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupe_tapis">
       <property name="title">
        <string>Tapis</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="AfficheurTapis" name="tapis">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <column>
           <property name="text">
            <string>Joueur</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Carte</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layout_maitre">
          <item>
           <widget class="QLabel" name="label_maitre">
            <property name="text">
             <string>Joueur maître :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="LabelNom" name="nom_maitre">
            <property name="text">
             <string>???</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupe_chien">
       <property name="title">
        <string>Chien</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="chien_1">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="chien_2">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="chien_3">
          <property name="text">
           <string>???</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="onglets">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="encheres">
      <attribute name="title">
       <string>Faire des enchères</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupe_enchere">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Choisissez votre enchère</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="6">
           <widget class="QRadioButton" name="bouton_hack_enchere">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Hack : </string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QRadioButton" name="bouton_garde_contre">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Garde Contre</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QRadioButton" name="bouton_garde_sans">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Garde Sans</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QSpinBox" name="selection_enchere">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QRadioButton" name="bouton_garde">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Garde</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="bouton_prise">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Prise</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="bouton_passe">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Passe</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="7">
           <widget class="QPushButton" name="bouton_enchere">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Faire cette enchère</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="appel">
      <attribute name="title">
       <string>Post-enchères</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Appelez une carte :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SelecteurCarte" name="selecteur_appel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="SelecteurCarte" name="ecart_1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SelecteurCarte" name="ecart_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SelecteurCarte" name="ecart_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bouton_appel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Appeler cette carte</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="bouton_ecart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Écarter ces cartes</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="jeu">
      <attribute name="title">
       <string>État du jeu</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Jeu d'une carte</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QRadioButton" name="bouton_hack_jeu">
            <property name="text">
             <string>Hack :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QRadioButton" name="bouton_possede">
            <property name="text">
             <string>Carte que je possède :</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="SelecteurCarte" name="cartes_jouables">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="bouton_requete">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Jouer</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QSpinBox" name="carte_hack">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox_7">
     <property name="title">
      <string>Journal</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="Journal" name="journal"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LabelNom</class>
   <extends>QLabel</extends>
   <header>label_nom.hpp</header>
  </customwidget>
  <customwidget>
   <class>Journal</class>
   <extends>QTextBrowser</extends>
   <header>journal.hpp</header>
  </customwidget>
  <customwidget>
   <class>AfficheurTapis</class>
   <extends>QTableWidget</extends>
   <header>afficheur_tapis.hpp</header>
  </customwidget>
  <customwidget>
   <class>ListeCartes</class>
   <extends>QTextBrowser</extends>
   <header>liste_cartes.hpp</header>
  </customwidget>
  <customwidget>
   <class>SelecteurCarte</class>
   <extends>QComboBox</extends>
   <header>selecteur_carte.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>bouton_hack_enchere</sender>
   <signal>toggled(bool)</signal>
   <receiver>selection_enchere</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>380</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_possede</sender>
   <signal>toggled(bool)</signal>
   <receiver>cartes_jouables</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bouton_hack_jeu</sender>
   <signal>toggled(bool)</signal>
   <receiver>carte_hack</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
